### ISSUE TITLE
Flow validation should not display a progress bar

### ISSUE THREAD

**Issue by tchiotludo** [kestra-io/kestra#1301]
The top n-progress bar is diplayed on every flow validation. 
We should hide this progress bar for flow validation

**Comment by loicmathieu** [comment]
I check and I don't think it's the case anymore.

Can you validate that you didn't see any progress bar anymore ?

**Comment by brian-mulier-p** [comment]
@loicmathieu It is displayed: 
![image](https://github.com/kestra-io/kestra/assets/37618489/85a68fc9-1c91-4650-ba70-0f9e86277d60)
It is an issue now that the flow validation occurs again after every input of the user (if he stops its input for 500ms).

**Comment by loicmathieu** [comment]
@brian-mulier-p you are right, I didn't notice it as in dark mode this bar is not very visible.
